pkg/apis/config/v1/types.go

```diff
@@ -0,0 +1,19 @@
+package v1
+
+type SippyConfig struct {
+	Prow     ProwConfig               `yaml:"prow"`
+	Releases map[string]ReleaseConfig `yaml:"releases"`
+}
+
+type ProwConfig struct {
+	// URL to the prowjob.js endpoint of the prow instance.
```

Comment:
Could you note some details on what this endpoint contains and how we use it.

Reply:
Done